--- ../pthreads-w32-2-8-0-release-orig/GNUmakefile	2006-04-30 12:22:33.000000000 +0200
+++ ./GNUmakefile	2008-07-12 12:20:10.000000000 +0200
@@ -49,7 +49,7 @@
 
 # For cross compiling use e.g.
 # make CROSS=i386-mingw32msvc- clean GC-inlined
-CROSS	= 
+CROSS	= mingw32-
 
 AR	= $(CROSS)ar
 DLLTOOL = $(CROSS)dlltool
--- ../pthreads-w32-2-8-0-release-orig/implement.h	2005-06-03 10:36:27.000000000 +0200
+++ ./implement.h	2008-07-12 13:42:01.000000000 +0200
@@ -77,7 +77,7 @@
 #include "sched.h"
 
 #if defined(HAVE_C_INLINE) || defined(__cplusplus)
-#define INLINE inline
+#define INLINE
 #else
 #define INLINE
 #endif
